docs/092-ht_change_voice.md

```diff
@@ -1,3 +1,47 @@
 # How To Change Your Assistant's Voice
 
-**WIP**
+## Using TTS server plugin
+
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "ovos-tts-plugin-server": {
+            "voice": "amy-low"
+        }
+    }
+}
+```
+
+Execute `systemctl --user restart ovos` to reload configuration.
```

Comment:
This is not always how you restart ovos.  raspOVOS uses systemd, but when running otherwise, it may not be the issue.

I would suggest making this "an example used with systemd services `systemctl --user restart ovos`"

Or something similar.  Point being, this is not the only way this is done.